### ISSUE TITLE
ProjectRed

### ISSUE THREAD

**Issue by Maineri** [MrTJP/ProjectRed#1235]
This new project red is just glitchy as fuck. Is it even under dev anymore. No updates in 4-5 months. And not really any progress here in github either.

**Comment by covers1624** [comment]
Tj Has just started school back up, and im waiting on his help to push 1.11 builds, then is on to the 1.12 port, from there on there will likely be more dev progress as i consider 1.11 and 1.10 un supported at this point, too many projects im not going to support across 3 different mc versions. If you could elaborate as to your "glitchy as fuck" issues that would be great.

**Comment by Maineri** [comment]
I dont even care what version this is for. I just want one stable version. And as I made post like half year ago , there is alot of white stuff everywhere. And somebody opens new issue every week but no build has been made since june so what do you think. Every reported issue since that is still in the game.

**Comment by MrTJP** [comment]
White stuff you say? I’ll get right on that...

**Comment by Maineri** [comment]
Well you can check my old post FROM 4 MONTHS AGO

**Comment by covers1624** [comment]
So, first off, being a dick and getting agrovated over this is not going to get you anywhere. Both Myself and TJ have lives and cant dedicate every second of it to modding. So, linking the issue would have been nice, https://github.com/MrTJP/ProjectRed/issues/1210  https://github.com/MrTJP/ProjectRed/issues/1208 https://github.com/MrTJP/ProjectRed/issues/1205. You provide no information on ANY of your issues about what version of PR you are running and no steps to reproduce them.
